examples/powershell/create-and-monitor-export-job.ps1

```diff
@@ -0,0 +1,103 @@
+###################################
+# create-and-monitor-export-job.ps1
+###################################
+
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+
+function Get-Bearer-Token {
```

Comment:
Is there any code here handling when retrieval fails with an explanation? So far analysts I have talked to appreciate hints about what may be wrong.

Reply:
Examples have been moved to different repos.